addons/payment_adyen/controllers/main.py

```diff
@@ -87,12 +87,16 @@ def adyen_payments(
         # Prepare the payment request to Adyen
         provider_sudo = request.env['payment.provider'].sudo().browse(provider_id).exists()
         tx_sudo = request.env['payment.transaction'].sudo().search([('reference', '=', reference)])
+        partner_country_code = (
+            tx_sudo.partner_country_id.code or provider_sudo.company_id.country_id.code or 'NL'
+        )
         data = {
             'merchantAccount': provider_sudo.adyen_merchant_account,
             'amount': {
                 'value': converted_amount,
                 'currency': request.env['res.currency'].browse(currency_id).name,  # ISO 4217
             },
+            'countryCode': partner_country_code,  # ISO 3166-1 alpha-2 (e.g.: 'BE')
```

Comment:
The `_send_payment_request` method also relies on the `/payments` endpoint; I assume we should do something about that one too?